saviSimulation/savi/jason_processing/UxVBehavior.java

```diff
@@ -5,14 +5,14 @@
 import processing.core.*;
 
 import processing.data.*;
-import processing.event.*; 
+import processing.event.*;
 import processing.opengl.*;
 import savi.StateSynchronization.*;
 
 
-public class UxVBehavior extends AgentModel {
-	private static final double SPEED = 0.1; // 0.1 pixels (whatever real-life distance this corresponds to)
-
+public abstract class UxVBehavior extends AgentModel {
+	protected static final double SPEED = 0.1; // 0.1 pixels (whatever real-life distance this corresponds to)
```

Comment:
Having SPEED here as static, we are forcing the UaV and UgV going at the same speed.
I would move the UaV and UgV speed to the config file or at least set different values for them.
This can be addressed in future PRs.

Reply:
Definitely should be in the config file.

Reply:
I agree, i'll add it on the next PR.

Reply:
👍